### PR TITLE
crowbar: Stop uploading proposal templates as chef data bags

### PR DESCRIPTION
**Pull request by vuntz**
We should not need them: the templates are only ever used when creating
proposals, and the rails app has direct access to the json file.

This should go with https://github.com/crowbar/crowbar/pull/2230

### CODE REVIEW

**Comment by nicolasbock** [comment]
Looks good, but waiting for gating tests.

File: crowbar_framework/app/models/network_service.rb [hunk 1]
```diff
@@ -256,8 +256,19 @@ def apply_role_pre_chef_call(old_role, role, all_nodes)
       db = Chef::DataBag.load("crowbar/#{k}_network") rescue nil
       if db.nil?
         @logger.debug("Network: creating #{k} in the network")
+
+        # ensure that crowbar data bag exists
+        databag_name = "crowbar"
+        begin
+          Chef::DataBag.load(databag_name)
+        rescue Net::HTTPServerException
+          crowbar_bar = Chef::DataBag.new
```
Comment (MaximilianMeister) [inline crowbar_framework/app/models/network_service.rb]: i guess you wanted to call it `crowbar_bag` ? looks like a typo

Comment (nicolasbock) [inline crowbar_framework/app/models/network_service.rb]: It's a double crowbar :smile:

**Comment by nicolasbock** [comment]
+1

File: crowbar_framework/app/models/network_service.rb (hunk 1 above)
Comment (vuntz) [inline crowbar_framework/app/models/network_service.rb]: @MaximilianMeister fixed

**Comment by rsalevsky** [comment]
+1

**Pull request merged**